client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { Button } from '@automattic/components';
+import { useBreakpoint } from '@automattic/viewport-react';
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent, useState } from 'react';
+import PromoCard from 'calypso/components/promo-section/promo-card';
+import PromoCardPrice from 'calypso/components/promo-section/promo-card/price';
+import EmailProviderFeaturesToggleButton from 'calypso/my-sites/email/email-provider-features/toggle-button';
+import EmailProviderStackedFeatures from 'calypso/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/email-provider-stacked-features';
+import { ProviderCard } from 'calypso/my-sites/email/email-providers-stacked-comparison/index';
+
+import './style.scss';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
+
+const EmailProvidersStackedCard: FunctionComponent< ProviderCard > = ( props ) => {
+	const {
+		additionalPriceInformation,
+		buttonLabel,
+		children,
+		description,
+		detailsExpanded,
+		discount,
+		expandButtonLabel,
+		features,
+		footerBadge,
+		formattedPrice,
+		formFields,
+		logo,
+		onButtonClick = noop,
+		onExpandedChange = noop,
+		productName,
+		providerKey,
+		showExpandButton = true,
+	} = props;
+
+	const translate = useTranslate();
+
+	const [ areFeaturesExpanded, setFeaturesExpanded ] = useState( false );
+
+	const isViewportSizeLowerThan1040px = useBreakpoint( '<1040px' );
+
+	const showFeaturesToggleButton = detailsExpanded && isViewportSizeLowerThan1040px;
+
+	const toggleVisibility = ( event: React.MouseEvent ): void => {
+		event.preventDefault();
+
+		onExpandedChange( providerKey, ! detailsExpanded );
+	};
+
+	const labelForExpandButton = expandButtonLabel ? expandButtonLabel : buttonLabel;
+
+	const price = (
+		<div className="email-provider-stacked-card__price-badge">
+			<div className="email-provider-stacked-card__discount badge badge--info-green">
+				{ translate( '3-Months free' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( '3 months free' )` *ES Score: 7*

Reply:
I believe it's either `3 months free` or `3-month free`. The current `Email Comparison` page shows `3 months free`.

Reply:
Oh I will change it, I used the string that was present in the mock up